bear/main.py.in

```diff
@@ -470,10 +493,22 @@ def create_intercept_parser():
         default=[os.getenv('CXX', 'c++')],
         help="""Hint '%(prog)s' to classify the given program name as C++
         compiler.""")
+    
+    #add_argument for Fortan
+    parser.add_argument(
+        '--use-f',
+        metavar='<path>',
+        dest='use_f',
+        action='append',
```

Comment:
The current default `fXX` does not look good. I don't think there is such compiler... Why not just using `f95`?